### PR TITLE
Totosprint add submodules

### PR DESCRIPTION
**Pull request by baloian**
Removed ssl_commons directory and created ssl_commons submodule.

Changed In-Toto old ssl_crypto submodule into the new ssl_crypto submodule, which is commons for
In-Toto and TuF.

Fixed unittest bugs

### CODE REVIEW

File: toto/settings.py [hunk 1]
```diff
@@ -65,6 +65,8 @@
 # 'pyca-cryptography']
 GENERAL_CRYPTO_LIBRARY = 'pycrypto'
 
+# Hash algorithms
+REPOSITORY_HASH_ALGORITHMS = ['sha256', 'sha512']
```
Comment (SantiagoTorres) [inline toto/settings.py:69]: Are these relevant to Toto?

**Review by lukpueh — CHANGES_REQUESTED** [review]
Thanks for the additions
- Please fix conflict in .gitmodules
- Please fix typos in commit message (2e7499)
- This PR moves the ssl_crypto submodule pointer from [542caed](https://github.com/secure-systems-lab/ssl_crypto/commit/542caedfc2c1d60f6d30ae1188aa68ad7cb7a389) to [ff19a50](https://github.com/secure-systems-lab/ssl_crypto/commit/ff19a508023cec859a3e2bc3a897030075743792) (9 commits). It would be nice to summarize those changes in this PR's description
- ssl_crypto@ff19a50 has failing unittests:
```
======================================================================
ERROR: test_create_signature (toto.ssl_crypto.tests.test_keys.TestKeys)
----------------------------------------------------------------------
ValueError: The required private key is unset.

======================================================================
ERROR: test_decrypt_key (toto.ssl_crypto.tests.test_keys.TestKeys)
----------------------------------------------------------------------
FormatError: Key object does not contain a private part.

======================================================================
ERROR: test_format_metadata_to_key (toto.ssl_crypto.tests.test_keys.TestKeys)
----------------------------------------------------------------------
FormatError: Wanted a u'RSAKEY_SCHEMA'.

======================================================================
ERROR: test_verify_signature (toto.ssl_crypto.tests.test_keys.TestKeys)
----------------------------------------------------------------------
FormatError: Missing key 'keyid' in u'ANYKEY_SCHEMA'
```

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,6 @@
+[submodule "toto/ssl_commons"]
+	path = toto/ssl_commons
+	url = https://github.com/secure-systems-lab/ssl_commons.git
 [submodule "toto/ssl_crypto"]
 	path = toto/ssl_crypto
-	url = git@github.com:/santiagotorres/ssl_crypto
+	url = https://github.com/secure-systems-lab/ssl_crypto.git
```
Comment (lukpueh) [inline .gitmodules]: Please use [spaces instead of tabs](https://github.com/secure-systems-lab/code-style-guidelines#tabs).

**Review by SantiagoTorres — CHANGES_REQUESTED** [review]

**Comment by lukpueh** [comment]
The architecture has changed https://github.com/in-toto/in-toto/issues/45#issuecomment-270733005, which makes above changes obsolete. Thanks for the work. I'll herby close this PR as "won't merge".

**Pull request closed without merging**